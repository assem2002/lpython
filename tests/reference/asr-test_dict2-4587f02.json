{
    "basename": "asr-test_dict2-4587f02",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_dict2.py",
    "infile_hash": "300adb7c040d5d2035f7da25a633f6f83b5adc55b572990ad4403a5b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_dict2-4587f02.stderr",
    "stderr_hash": "291c5956c4006303abdbe6d41bcac8de6c0a967a04a4747ef6a5e456",
    "returncode": 2
}