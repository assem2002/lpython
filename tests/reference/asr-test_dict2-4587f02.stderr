semantic error: Key type should be 'str' instead of 'i32'
 --> tests/errors/test_dict2.py:4:7
  |
4 |     y[1] = -3
  |       ^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
